DotNetCLI/Get-VisualStudioSolutionProjectPaths/Get-VisualStudioSolutionProjectPaths.psm1

```diff
@@ -1,4 +1,4 @@
-function Get-VisualStudioSolutionProjectPaths
+﻿function Get-VisualStudioSolutionProjectPath
```

Comment:
`Get-VisualStudioSolutionProjectsPath` (project becomes plural) instead? Signals the intention more clearly.

Reply:
The same point applies here, the logic here is like an advanced `Get-ChildItem` as well.

(And now I start dreaming about a world here `Get-ChildItem` has a `-Type` parameter just like `New-Item`, and we could extend it with a custom `VisualStudioSolutionProject` type. That would be so neat...)